h2o-core/src/main/java/water/MRTask.java

```diff
@@ -252,7 +252,8 @@ public void map( Chunk c, NewChunk nc ) { }
    *  <strong>local</strong> Chunks.  All map variants are called, but only one
    *  is expected to be overridden. */
   public void map( Chunk c0, Chunk c1 ) { }
-  //public void map( Chunk c0, Chunk c1, NewChunk nc) { }
```

Comment:
They were disabled since we did not expect users to use them (i.e., they were waiting for use-case)